src/devices/r900.c

```diff
@@ -0,0 +1,297 @@
+/** @file
+    Neptune r900 protocol decoder for r900 based flow meters, tested with my water meter.
```

Comment:
This line needs to be line 11 (first in next doc comment).
And here it should just be "Neptune R900 flow meter decoder."